File: scripts/pi2go/Display1.py
# Display1.py

from raspibrick import *

robot = Robot()
display = Display()
display.showText("1234")
Tools.delay(3000)
display.showText("1234567890")
Tools.delay(3000)
display.showText("1234567890", pos = 2)
Tools.delay(3000)
display.showText("1234567890", pos = -2)
Tools.delay(3000)
display.showText("1234567890", pos = 1, dp = [1, 1, 0, 0])
Tools.delay(3000)
display.showText("1234567890", pos = 1, dp = [0, 1])
Tools.delay(3000)
robot.exit()
print "All done"

File: programs/Display1.java
// Display1.java

import ch.aplu.raspi.*;

public class Display1
{
  public Display1()
  {
    Robot robot = new Robot();
    Display display = new Display();
    for (int digit = 0; digit < 4; digit++)
    {
      display.setDigit('A', digit);
      Tools.delay(1000);
    }
    robot.exit();
  }

  public static void main(String[] args)
  {
    new Display1();
  }
}
